packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -137,6 +141,7 @@ const defaultProps = {
   onChange: undefined,
   onChangeRaw: undefined,
   onClickOutside: undefined,
+  onDismiss: undefined,
```

Comment:
this is undefined by default

Reply:
Removed it here https://github.com/cerner/terra-framework/pull/1224/commits/22d489afc2983bcb40c258881a6ed2beab768874